micro_ros_setup/config/freertos/nucleo_f446ze/board.repos

```diff
@@ -0,0 +1,5 @@
+repositories:
+  freertos_apps:
+    type: git
+    url: https://github.com/alsaibie/freertos_apps
```

Comment:
Hello @alsaibie please the this URL to the correct link in the already merged repo: https://github.com/micro-ROS/freertos_apps

Reply:
Done.

Reply:
Thanks!